### DISCUSSION TITLE
Time for changes to take effect?

### DISCUSSION THREAD

**Discussion by Vaikevimbu** [electricitymaps/electricitymaps-contrib#2810] [Q&A] [answered]
I made a pull request few weeks ago to update Estonian production capacities. It was approved and merged more than a week ago, but the map is still showing old capacities for Estonia. How much time is normal for the changes to take effect? I am wondering if something more needs to be done.

**Comment by Kongkille** [comment, marked as answer]
Hi yes, we need to bump it as well. I'll try to get it done asap!

However, you raise a good point, perhaps we should setup some recurring timeslots for bumping, so people know when their changes are going to go live

**Reply by VIKTORVAV99** [reply]
I believe this is still relevant and a weekly release on let's say Mondays (just don't do Fridays or the weekend) would be nice.
(this should obviously not exclude bug fixes and reversions from getting "emergency releases")

**Reply by Kongkille** [reply]
That is a good point. I will make a calendar notification weekly to get the latest changes out
